frontend/src/queries/nodes/DataTable/utils.ts

```diff
@@ -46,3 +46,10 @@ export function removeExpressionComment(query: string): string {
     }
     return query.trim()
 }
+
+export function trimQuotes(identifier: string): string {
+    if (identifier.startsWith('"') && identifier.endsWith('"')) {
+        return identifier.slice(1, -1)
+    }
```

Comment:
Let's cover the case of backquotes as well.

```suggestion
    if ((identifier.startsWith('"') || identifier.endsWith('`')) && identifier.endsWith(identifier[0])) {
        return identifier.slice(1, -1)
    }
```